docs/chunks-storage/table-manager.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Table-manager"
+linkTitle: "Table-manager"
+weight: 3
+slug: table-manager
+---
+
+The table-manager is the Cortex service responsible for creating the [periodic tables](./schema-config.md) used to store index and chunks, and deleting them once their data time range exceeds the retention period (if retention is enabled).
+
+_For more information about the schema config and periodic tables, please refer to the [Schema config](./schema-config.md) documentation._
+
+## Table creation
+
+The table-manager creates new tables slightly ahead of their start period, in order to make sure that the new table is ready once the current table end period is reached. The `-table-manager.periodic-table.grace-period` config option defines how long before a table should be created.
+
+## Retention
+
+The retention - managed by the table-manager - is **disabled by default**, due to its destructive nature. You can enable the data retention explicitly via `-table-manager.retention-deletes-enabled=true` and setting `-table-manager.retention-period` to a value greater than zero.
+
+The table-manager implements the retention deleting the entire tables whose data exceeded the retention period. This design allows to have fast delete operations, at the cost of having a retention granularity controlled by the table's [`period`](./schema-config.md#schema-config).
+
+Given each table contains data for `period` of time and that the entire table is deleted, the table-manager keeps the last tables alive using this formula:
+
+```
+number_of_tables_to_keep = floor(retention_period / table_period) + 1
+```
+
+![Table-manager retention](/images/chunks-storage/table-manager-retention.png)
```

Comment:
Love the images, but can we link the sources somewhere so people in the future can edit them if required.

Reply:
Good point, done!